test/unittests/unittest.cpp

```diff
@@ -22,25 +22,36 @@
 *
 * Author: Ken Zangelin
 */
-#include "rest/uriParamNames.h"
+#include <string>
+#include <vector>
+#include <map>
 
-#include "unittest.h"
-#include "testInit.h"
 #include "cache/subCache.h"
+#include "rest/uriParamNames.h"
+
+#include "unittests/unittest.h"
+#include "unittests/testInit.h"
+
+
+
+/* ****************************************************************************
+*
+* USING
+*/
+using ::testing::Return;
 
 
 
 /* ****************************************************************************
 *
-* debugging - 
+* debugging -
 *
 * FIXME P4 - these counters are useful (only visible if UT_DEBUG is defined)
 *            to detect calls to utInit without respective call to utExit.
 *            All of it should be removed as soon as we implement the new
 *            more complete TRST macro (I_TEST?)
-* 
 */
-//#define UT_DEBUG
+// #define UT_DEBUG
```

Comment:
No, this is old.  I only added the space, as the style guide asks for it
This line is to be uncommented to run in debug mode